src/fmu/dataio/_model/specification.py

```diff
@@ -52,7 +52,7 @@ class SurfaceSpecification(RowColumn):
 class PointSpecification(BaseModel):
     """Specifies relevant values describing an xyz points object."""
 
-    attributes: Optional[List[str]]
+    attributes: Optional[List[str]] = Field(default_factory=list)
```

Comment:
This needs a closer look?

Reply:
Yes default=None might be better here

Reply:
And most likely a pattern we have elsewhere

Reply:
Changed to None, maybe I am seeing things 🧐